pkg/systemd/overview-cards/realmd.jsx

```diff
@@ -112,6 +115,8 @@
     }
 
     checkRealm(name) {
+        if (!this.dbus_realmd)
+            return Promise.reject();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18240-20230130-105647-26c41b66-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)